src/util/interaction/out.ts

```diff
@@ -116,14 +118,26 @@ export function table(...args: any[]): void {
 // Formatting helper for cli-table2 - no table outlines. Used by
 // help commands for formatting lists of options, commands, etc.
 //
-export const noTableBorders = {
-  chars: {
-    "top": "", "top-mid": "", "top-left": "", "top-right": "",
-    "bottom": "", "bottom-mid": "", "bottom-left": "", "bottom-right": "",
-    "left": "", "left-mid": "", "mid": "", "mid-mid": "",
-    "right": "", "right-mid": "", "middle": " "
-  },
-  style: { "padding-left": 0, "padding-right": 0 }
+export function getNoTableBordersSetting(tableColumnsCount: number) {
+  let consoleWidth = terminal.columns();
+  // There is a single whitespace to the right from the each column, count it as unavailable
+  let availableWidth = consoleWidth - tableColumnsCount;
+  let colWidth = Math.floor(availableWidth / tableColumnsCount);
+  let remainderWidth = availableWidth % tableColumnsCount;
+  // Spreading remainder width across the first columns
+  let colWidths = Array<number>(remainderWidth).fill(colWidth + 1).concat(Array<number>(tableColumnsCount - remainderWidth).fill(colWidth));
```

Comment:
Also, I'm fine leaving this function hard-wired for the help system at the moment.